packages/cli-tools/bin/streamr-stream-subscribe.ts

```diff
@@ -1,19 +1,26 @@
 #!/usr/bin/env node
 import '../src/logLevel'
-import StreamrClient from 'streamr-client'
+import omit from 'lodash/omit'
+import StreamrClient, { MessageMetadata } from 'streamr-client'
 import { createClientCommand, Options as BaseOptions } from '../src/command'
 import { createFnParseInt } from '../src/common'
 
 interface Options extends BaseOptions {
     partition: number
     disableOrdering: boolean
+    raw: boolean
+    withMetadata: boolean
 }
 
 createClientCommand(async (client: StreamrClient, streamId: string, options: Options) => {
+    const formMessage = options.withMetadata
+        ? (message: unknown, metadata: MessageMetadata) => ({ message, metadata: omit(metadata, 'streamMessage') })
+        : (message: unknown) => message
     await client.subscribe({
         streamId,
         partition: options.partition,
-    }, (message) => console.info(JSON.stringify(message)))
+        raw: options.raw
+    }, (message, metadata) => console.info(JSON.stringify(formMessage(message, metadata))))
```

Comment:
Do not stringify the hex output of encrypted message when using raw?

Reply:
fixed